### PR TITLE
Bump pkgrel and explain change (follow up from #738)

### PR DESCRIPTION
**Pull request by PabloCastellano**

### CODE REVIEW

File: aports/device/linux-samsung-i9100/APKBUILD [hunk 1]
```diff
@@ -12,6 +12,7 @@
 # - enable devtmpfs (needed for udev -> touch support in weston)
 # - change compression from CONFIG_KERNEL_LZMA=y to
 #	CONFIG_KERNEL_GZIP=y (so it works with busybox)
+#	CONFIG_ANDROID_PARANOID_NETWORK=n (removed network restrictions)
```
Comment (z3ntu) [inline aports/device/linux-samsung-i9100/APKBUILD]: Actually in the file it doesn't get set to "n" but to "unset"

Comment (z3ntu) [reply aports/device/linux-samsung-i9100/APKBUILD]: Also the format is wrong, should be
```
# - CONFIG_ANDROID_PARANOID_NETWORK=n (removed network restrictions)
```

File: aports/device/linux-samsung-i9100/APKBUILD [hunk 2]
```diff
@@ -11,7 +11,8 @@
 # Kernel config changes, based on: arch/arm/configs/i9100_defconfig
 # - enable devtmpfs (needed for udev -> touch support in weston)
 # - change compression from CONFIG_KERNEL_LZMA=y to
-#	CONFIG_KERNEL_GZIP=y (so it works with busybox)
+# - set CONFIG_KERNEL_GZIP=y (so it works with busybox)
```
Comment (z3ntu) [inline aports/device/linux-samsung-i9100/APKBUILD]: nah that line is good, it (basically) says "change LZMA to GZIP"

File: aports/device/linux-samsung-i9100/APKBUILD [hunk 3]
```diff
@@ -11,7 +11,8 @@
...
+# - unset CONFIG_ANDROID_PARANOID_NETWORK (remove network restrictions)
```
Comment (z3ntu) [inline aports/device/linux-samsung-i9100/APKBUILD]: I've used `disable ANDROID_PARANOID_NETWORK (removes network restrictions)` in #742 , dunno what is better

**Comment by PabloCastellano** [comment]
Changed again. This PR must be the silliest one with more reviews ever :rofl:

File: aports/device/linux-samsung-i9100/APKBUILD [hunk 4]
```diff
@@ -11,7 +11,8 @@
 # Kernel config changes, based on: arch/arm/configs/i9100_defconfig
 # - enable devtmpfs (needed for udev -> touch support in weston)
 # - change compression from CONFIG_KERNEL_LZMA=y to
-#	CONFIG_KERNEL_GZIP=y (so it works with busybox)
+# 	CONFIG_KERNEL_GZIP=y (so it works with busybox)
```
Comment (z3ntu) [inline aports/device/linux-samsung-i9100/APKBUILD]: Probably BUT THERE'S AN EXTRA SPACE NOW (i don't really care but yeah 😆 )

**Comment by bhuvi8** [comment]
Thank you for taking this up. It was stupid on my part to not update the pkgrel and add an explanation

**Pull request merged**